packages/core/admin/admin/src/components/AuthenticatedApp/utils/api.js

```diff
@@ -9,10 +8,8 @@ const { get } = getFetchClient();
 
 const fetchStrapiLatestRelease = async (toggleNotification) => {
   try {
-    const {
-      data: { tag_name },
-    } = await axios.get('https://api.github.com/repos/strapi/strapi/releases/latest');
-
+    const { tag_name } = await fetch('https://api.github.com/repos/strapi/strapi/releases/latest').then(res => res.json()).catch(() => {});
```

Comment:
This is not quite what I meant :) Everything is wrapped in a `try / catch` and before `axios.get` would have thrown an error if the request fails.

If you want to keep the `try / catch` block you need to throw an error:

```
try {
  const res = await fetch();
  
  if (!res.ok) {
    throw new Error();
  }

  const data = await res.json();
  
  
} catch(err) {}
```

Right now you would swallow the error and `checkLatestStrapiVersion` might run into an error, because `tag_name` is not defined. 

I can recommend https://youtu.be/qH_WsIZ23YU?t=202

Reply:
I have refactored the code

Reply:
ok thanks